testcontainers/src/images/local_stack.rs

```diff
@@ -0,0 +1,72 @@
+use crate::{core::WaitFor, Image};
+use std::collections::HashMap;
+
+const NAME: &str = "localstack/localstack";
+const TAG: &str = "latest";
+const PORT: u16 = 4566;
+
+#[derive(Debug)]
+pub struct LocalStack {
+    env_vars: HashMap<String, String>,
+    volumes: HashMap<String, String>,
+    tag: String,
+}
+
+impl LocalStack{
+    pub fn set_services(&mut self, services: &str) {
+
+        self.env_vars.insert("SERVICES".to_owned(), services.to_owned());
+
+    }
+}
+
+impl Default for LocalStack {
+    fn default() -> Self {
+        let mut env_vars = HashMap::new();
+        //env_vars.insert("SERVICES".to_owned(), "s3".to_owned());
+        env_vars.insert("DEBUG".to_owned(), "1".to_owned());
+        env_vars.insert("PORT_WEB_UI".to_owned(), "8080".to_owned());
+        env_vars.insert("LAMBDA_EXECUTOR".to_owned(), "docker".to_owned());
+        env_vars.insert("DOCKER_HOST".to_owned(), "unix:///var/run/docker.sock".to_owned());
+        env_vars.insert("AWS_DEFAULT_REGION".to_owned(), "eu-central-1".to_owned());
+        env_vars.insert("AWS_ACCESS_KEY_ID".to_owned(), "test".to_owned());
+        env_vars.insert("AWS_SECRET_ACCESS_KEY".to_owned(), "test".to_owned());
+        env_vars.insert("DATA_DIR".to_owned(), "/tmp/localstack/data".to_owned());
+
+        let mut volumes = HashMap::new();
+        volumes.insert("/var/run/docker.sock".to_owned(), "/var/run/docker.sock".to_owned());
+        LocalStack {
+            env_vars,
+            tag: TAG.to_owned(),
+            volumes
+        }
+    }
+}
+
+impl Image for LocalStack {
+    type Args = ();
+
+    fn name(&self) -> String {
+        NAME.to_owned()
+    }
+
+    fn tag(&self) -> String {
+        self.tag.to_owned()
+    }
+
+    fn ready_conditions(&self) -> Vec<WaitFor> {
+        vec![WaitFor::message_on_stdout( format!("Ready."))]
```

Comment:
Recent versions should use the health check endpoint `/_localstack/health` to detect readiness of the container.

Reply:
ok, but it'd requiere more effort parsing the /health response, right? or do you suggest to check it it reponses with HTTP 200/OK?

Reply:
Yes, we check for HTTP 200/OK.

Reply:
I'm afraid my knowledge of the Rust implementation is limited. Are you saying there is no wait strategy available to wait for an HTTP status code? Perhaps we could use `WaitFor::Healthcheck` or add the strategy to Rust as a new feature. That would be really useful.